### ISSUE TITLE
[Discussion] Move styled-media-query package to the SC organization 

### ISSUE THREAD

**Issue by morajabi** [styled-components/styled-components#1503]
TL;DR: I'd like to hear what do you think about moving `styled-media-query` to the organization? 

I know media queries are trivial in styled-components, also a simple utility function can be a media query, but some people rather to not write their own utility functions.

Its API is quite readable compared to normal confusing `max-width` and `min-width` and it converts units (px to rem, em to px, etc) with built-in utils.

https://github.com/morajabi/styled-media-query/

For the next version, I may make use of `theme` for breakpoints and change the API a little bit.

Let me know what do you think about moving this, how it can be better, or why you hate/love it! 😉

/cc @styled-components/core

**Comment by mxstbr** [comment]
I feel like that's a package that's perfectly fine to live in your personal GitHub—there's nothing really to be gained from moving it to the organization imo. :blush:

**Comment by morajabi** [comment]
At the time of writing the first paragraph of this issue, I felt like you, but wrote it anyway, what can go wrong! Thanks :)
